integration_tests/mockApis/csipApi.ts

```diff
@@ -537,6 +498,85 @@ const stubPostInvestigation = () => {
   })
 }
 
+const referralSubmittedCsip = {
+  recordUuid: '02e5854f-f7b1-4c56-bec8-69e390eb8550',
+  prisonNumber: 'A1111AA',
+  prisonCodeWhenRecorded: 'LEI',
+  createdAt: '2024-07-22T11:21:48',
```

Comment:
we might have `screeningSubmittedCsip` `investigationSubmittedCsip` `decisionSubmittedCsip` etc similar to this, since each of them will just be the same object with an extra property populated?

Reply:
I've just been using the same json and overriding the status as seen further up. Perhaps better to rename this to just `csip`? I dont want to create lots of variables that are just for single use.